src/interfaces/messages/handlers/messages-get.ts

```diff
@@ -62,7 +63,9 @@ export class MessagesGetHandler implements MethodHandler {
       }
 
       const { interface: messageInterface, method } = message.descriptor;
-      if (messageInterface !== DwnInterfaceName.Records || method !== DwnMethodName.Write) {
+      const interfaceMethod = `${messageInterface}${method}`;
+
+      if (interfaceMethod !== RecordsWrite) {
```

Comment:
im confused.  what was wrong with the old logic?

Reply:
just feel like it's easier to read

Reply:
@mistermoe ngl disagree, but not gonna stop you.

having a var named `RecordsWrite` is confusing cuz dwn-sdk-js has a class named `RecordsWrite` already.
if we really wanna make this more readable, the ruby-ish thing that I would do is define a bunch of methods like `isRecordsWrite(message)`. might be overkill, but i tend to like that specific type of overkill.

Reply:
ah.  personally i find this less easy to read, especially since there's already a `class RecordsWrite` elsewhere in this repo (tho we could have a `RecordsWrite.Identifier` or something?) 😕

perhaps we could flip this around and have it be `if (interface === DwnInterfaceName.Records && method === DwnMethodName.Write)` (or put a `if (!(...))` if you wanna keep it with `continue`)?

Reply:
the crowd has spoken! reverting

Reply:
what are y'all talking about? i didn't change anything 🥸